src/models/index.ts

```diff
@@ -79,6 +79,7 @@ export { VerticalSearchResponse } from './searchservice/response/VerticalSearchR
 export { Snippet } from './searchservice/response/Snippet';
 export { ErrorType } from './searchservice/response/ErrorType';
 export { FailedVertical } from './searchservice/response/FailedVertical';
+export { EmailFieldValueDirectAnswer } from './searchservice/response/EmailFieldValueDirectAnswer';
```

Comment:
nit: can we update this branch with the changes in develop and add this to where the other FieldValue interfaces are in the file?